src/System.Diagnostics.FileVersionInfo/src/System.Diagnostics.FileVersionInfo.builds

```diff
@@ -9,6 +9,11 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Diagnostics.FileVersionInfo.csproj">
+      <OSGroup>Windows_NT</OSGroup>
+      <TargetGroup>netcore50</TargetGroup>
```

Comment:
Ideally we'd do both :)  But certainly the latter should be quick and general goodness.